module-telemetry/src/main/java/com/mapbox/maps/module/telemetry/MapTelemetryImpl.kt

```diff
@@ -100,8 +135,10 @@ class MapTelemetryImpl : MapTelemetry {
    * @param interval the selected session interval
    * @return true if rotation session id was updated
    */
+  @Deprecated("setSessionIdRotationInterval has been deprecated and will do no operations")
```

Comment:
Remove comment except for the Deprecated note?

Reply:
imo we shouldn't remove doc even its deprecated, but maybe state the API is deprecated in the API doc to make it clear. We also have checks to validate all the public APIs have documentation..

Reply:
Well what's the benefit of having some description and then saying method is no-op? IMO will only confuse users. Just having a short one saying `No-up, deprecated, do not use this method` will have a clear indication that this method should not be used